### DISCUSSION TITLE
Can you suppress "For further information visit..." with ValidationError?

### DISCUSSION THREAD

**Discussion by TylerGrantSmith** [pydantic/pydantic#6954] [Question] [answered]
Can you suppress "For further information visit..." with ValidationError?

**Comment by Kludex** [comment, marked as answer]
You can. You need to catch the `ValidationError`, and do what it's explained here: https://docs.pydantic.dev/latest/errors/errors/#customize-error-messages. 🙏

**Comment by HansBambel** [comment]
I also find it a bit overkill to add a link for every validation error. It should be removable.

**Comment by hfudev** [comment]
That would be great if `pydantic` could support it in the `ConfigDict`.

**Comment by fortzi** [comment]
It would be great to just drop this line instead of having to completely redefine the formatting for the entire error message
